.github/workflows/smoke-test.yml

```diff
@@ -0,0 +1,21 @@
+name: Smoke test
+
+on:
+  schedule:
+    - cron: '0 0 * * SUN'
+  workflow_dispatch:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 14
```

Comment:
```suggestion
```

Let's remove this, use the latest LTS

Reply:
What's the syntax for `LTS`? By removing this the CI will default to some old version.

```
Expected version ">=12.11". Got "10.23.0"
```
https://github.com/AriPerkkio/eslint-plugin-unicorn/runs/1509597287?check_suite_focus=true

Reply:
This seems to work now with `actions/setup-node@v2-beta`.

Reply:
Do we need care about Node.js version?

If not let's use whatever this action decided, so we don't need update it in future.

Reply:
Minimum requirement is `12.11` which introduced some new features to `worker_threads`, required by `eslint-remote-tester`. 

The v14 was originally copied from integration step:

https://github.com/sindresorhus/eslint-plugin-unicorn/blob/d669707d18609b9b4672c7aa768076d01d671948/.github/workflows/main.yml#L50-L54

Reply:
Okay, add it back.